.gitignore

```diff
@@ -57,4 +57,7 @@ bin/
 [Bb][Ii][Nn]/
 
 # ignore googletest items
-test/googletest-*/
\ No newline at end of file
+test/googletest-*/
```

Comment:
This needs to be changed to catch all of the files generated by google test(see main review comment)
You should also make sure you git rid of any executable files that are being generated.

You may need to delete them so they are no longer being tracked.